benchmarks/graphics_pipeline/main.cpp

```diff
@@ -123,6 +123,10 @@ class ProjApp
 private:
     std::shared_ptr<KnobDropdown<std::string>> pKnobVs;
     std::shared_ptr<KnobDropdown<std::string>> pKnobPs;
+    std::shared_ptr<KnobCheckbox>              pKnobPlaceholder1;
```

Comment:
imo, usage examples are should be in tests,  not in actual benchmark/application.